x-pack/legacy/plugins/security/public/hacks/on_unauthorized_response.js

```diff
@@ -1,38 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-
-import { identity } from 'lodash';
-import { uiModules } from 'ui/modules';
-import { Path } from 'plugins/xpack_main/services/path';
-import 'plugins/security/services/auto_logout';
-
-function isUnauthorizedResponseAllowed(response) {
-  const API_WHITELIST = ['/internal/security/login', '/internal/security/users/.*/password'];
```

Comment:
__note:__ these are handled using NP HTTP client now and hence handled by NP interceptors. Moreover we return `403` if change password API call fails because of wrong `existing password` field so we shouldn't whitelist it anymore it seems.